Makefile

```diff
@@ -35,9 +35,22 @@ CXX = g++
 endif
 BENDER_VERSION = 0.21.0
 
-# We need a recent LLVM installation (>11) to compile Verilator
-CLANG_CC  ?= clang
-CLANG_CXX ?= clang++
+# We need a recent LLVM installation (>11) to compile Verilator.
+# We also need to link the binaries with LLVM's libc++.
+# Define CLANG_PATH to be the path of your Clang installation.
+# At IIS, check .gitlab/.gitlab-ci.yml for an example CLANG_PATH.
+
+ifneq (${CLANG_PATH},)
+        CLANG_CC       := $(CLANG_PATH)/bin/clang
```

Comment:
If you define `CLANG_PATH`, then you do not need to specify `CLANC_CC`. What do you think about this solution?

Reply:
Perfect.

Just a small consistency question. How many spaces are we using for this kind of indentation? Here you use eight, above we have zero. Personally, I think if we want to indent the blocks to make them more readable, we should use two spaces as we do in all the other source files.

Reply:
It is not consistent at all. In the `hardware`'s Makefile, we indent with two spaces.